examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-5_Privacy_In_Federated_Learning/05.3_homomorphic_encryption/requirements.txt

```diff
@@ -0,0 +1,7 @@
+nvflare[HE]~=2.5
```

Comment:
we need to update these to 2.6 later

Reply:
I changed it to be >=2.5 as the examples in this chapter will also work with 2.5.